app/src/myanmar/java/org/eyeseetea/malariacare/database/PostMigration.java

```diff
@@ -12,4 +12,5 @@
     public static void launchPostMigration() {
         Migration15AddOptionAttributeColumns.postMigrate();
     }
+
```

Comment:
@gnarbaiz why this space? If you're not doing anything else in this file, remember 2 things:
- 1) making changes to any file increases the risk of conflict with other developers, so unless you really want to make a change, please don't do it
- 2) you have to review every file added for commit before committing them and even more before uploading them to github repository. Please only submit what's part of your solution